src/internal/handle_modal.go

```
@@ -25,6 +25,11 @@ func (m *model) createItem() {
 		m.typingModal.open = false
 		m.typingModal.textInput.Blur()
 	}()
+
+	if isValid := isValidFileName(m.typingModal.textInput.Value()); !isValid {
+		return
```

Silently failing is not good. We need to log the error via slog.Error. 

But we also need to give some kind of feedback to the user interface.

Maybe print a line in the typingModal.
Add a field `errorMessage` in `m.typingModal`. Print that error message (if not empty) in render
- See `typineModalRender()` in `model_render.go`